docs/sources/operations/request-validation-rate-limits.md

```diff
@@ -0,0 +1,226 @@
+
+This value can be modified globally in the [`limits_config`](https://grafana.com/docs/loki/latest/configuration/#limits_config) block, or on a per-tenant basis in the [runtime overrides](https://grafana.com/docs/loki/latest/configuration/#runtime-configuration-file) file. This error can be solved by increasing the `creation_grace_period` value, or investigating why this particular stream has a timestamp too far into the future. The stream in question will be returned in the body of the HTTP response.
+
+| Property                | Value             |
+|-------------------------|-------------------|
+| Enforced by             | `distributor`     |
+| Outcome                 | Request rejected  |
+| Retryable               | **No**            |
+| Sample discarded        | **Yes**           |
+| Configurable per tenant | Yes               |
+| HTTP status code        | `400 Bad Request` |
+
+## `max_label_names_per_series`
+
+If a series is submitted with more labels than Loki has been configured to allow, this error will occur.
+
+This value can be modified globally in the [`limits_config`](https://grafana.com/docs/loki/latest/configuration/#limits_config) block, or on a per-tenant basis in the [runtime overrides](https://grafana.com/docs/loki/latest/configuration/#runtime-configuration-file) file. This error can be solved by increasing the `max_label_names_per_series` value. The offending stream will be returned in the body of the HTTP response.
+
+| Property                | Value             |
+|-------------------------|-------------------|
+| Enforced by             | `distributor`     |
+| Outcome                 | Request rejected  |
+| Retryable               | **No**            |
+| Sample discarded        | **Yes**           |
+| Configurable per tenant | Yes               |
+| HTTP status code        | `400 Bad Request` |
+
+## `label_name_too_long`
+
+If a series has a label name with a length in bytes greater than Loki has been configured to allow, this error will occur.
+
+This value can be modified globally in the [`limits_config`](https://grafana.com/docs/loki/latest/configuration/#limits_config) block, or on a per-tenant basis in the [runtime overrides](https://grafana.com/docs/loki/latest/configuration/#runtime-configuration-file) file. This error can be solved by increasing the `max_label_name_length` value. The offending stream will be returned in the body of the HTTP response.
+
+| Property                | Value             |
+|-------------------------|-------------------|
+| Enforced by             | `distributor`     |
+| Outcome                 | Request rejected  |
+| Retryable               | **No**            |
+| Sample discarded        | **Yes**           |
+| Configurable per tenant | Yes               |
+| HTTP status code        | `400 Bad Request` |
+
+## `label_value_too_long`
+
+If a series has a label value with a length in bytes greater than Loki has been configured to allow, this error will occur.
```

Comment:
```suggestion
If a sample has a label value with a length in bytes greater than Loki has been configured to allow, it will be rejected for `reason=label_value_too_long`. 
```